rewriter/ChainedSig.cc

```diff
@@ -0,0 +1,40 @@
+#include "rewriter/ChainedSig.h"
+#include "ast/Helpers.h"
+#include "ast/treemap/treemap.h"
+
+using namespace std;
+namespace sorbet::rewriter {
+
+// Rewrite the new chained syntax into the old syntax. E.g.:
+// sig.abstract { void } is rewritten as sig { abstract.void }
+// Only tries to rewrite if the current send function is one of the accepted methods
+// and the receiver is sig.
```

Comment:
Can you move this comment to the header file, and structure it similar to the header comments on other rewriter passes?

For example

https://github.com/sorbet/sorbet/blob/a9fb984be106118b7f5b757c29f09b70993e8792/rewriter/Prop.h#L7-L25